lnwallet/channel.go

```diff
@@ -3711,6 +3711,16 @@ func (lc *LightningChannel) ReceiveNewCommitment(commitSig *btcec.Signature,
 	// our local commitment chain.
 	localCommitmentView.sig = commitSig.Serialize()
 	lc.localCommitChain.addCommitment(localCommitmentView)
+	// TODO: remove -------------------
```

Comment:
This is just added for now to show that we sign a state with a negative balance.